### ISSUE TITLE
bun server exist with SEGFAULT signal when upload file.

### ISSUE THREAD

**Issue by robobun** [oven-sh/bun#5171]
I tried the following code:

```js
import { Elysia, t } from "elysia";

const app = new Elysia()
  .post("/", async ({ body: { data } }) => data, {
    body: t.Object({
      data: t.File({ maxSize: 100000 }),
    }),
  })
  .listen(8000);

console.log(`app is running at ${app.server?.hostname}:${app.server?.port}`);
```

When i use postmen to test this upload file api, the server get crash when I upload some kind of file such as image, pdf.

_Originally reported on Discord: [`bun server exist with SEGFAULT signal when upload file.`](https://discord.com/channels/876711213126520882/1151087773135147098/1151087773135147098)_

**Comment by akx** [comment]
Probably a duplicate of #4966.

**Comment by Electroid** [comment]
Should be fixed in  #5152
